homeassistant/components/verisure/coordinator.py

```diff
@@ -81,13 +76,6 @@ async def _async_update_data(self) -> dict:
                 self.verisure.smart_lock(),
                 self.verisure.smartplugs(),
             )
-        except VerisureResponseError as err:
-            LOGGER.debug("Cookie expired or service unavailable, %s", err)
-            overview = self._overview
-            try:
-                await self.hass.async_add_executor_job(self.verisure.update_cookie)
-            except VerisureResponseError as ex:
-                raise ConfigEntryAuthFailed("Credentials for Verisure expired.") from ex
```

Comment:
Same?

Reply:
As in, same question as [before](https://github.com/home-assistant/core/pull/97278#discussion_r1275382248)?

Reply:
yup, we need to handle auth expiry, I'm not surely how that works with removing this?

Reply:
reverted